test/functional/tests/common/lvm_common_utils.py

```diff
@@ -1,13 +1,14 @@
 #
-# Copyright(c) 2021 Intel Corporation
+# Copyright(c) 2022 Intel Corporation
 # SPDX-License-Identifier: BSD-3-Clause
```

Comment:
looks like you have two changes in one comit: introducing LVM and changes in get_test_configuration(). I suggest to split this into two commits.